receiver/k8sobjectsreceiver/testdata/invalid_config.yaml

```diff
@@ -0,0 +1,18 @@
+receivers:
+  k8sobjects/invalid_resource:
+    objects:
+      - name: fake_resource
+        mode: watch
+
+processors:
+  nop:
+
+exporters:
+  nop:
+
+service:
+  pipelines:
+    logs:
+      receivers: [k8sobjects]
+      processors: [nop]
+      exporters: [nop]
```

Comment:
Don't need to include pipelines in the tests. please follow the same patterns as done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14704